internal/internal_worker.go

```diff
@@ -1235,7 +1235,7 @@ func initBinaryChecksumLocked() error {
 		_ = f.Close() // error is unimportant as it is read-only
 	}()
 
-	h := md5.New()
+	h := sha256.New()
```

Comment:
Doubles the checksum size, but I don't think that's a big deal, nor do I think performance switching from md5 to sha256 is a concern